app/qml/MerginProjectPanel.qml

```diff
@@ -56,7 +55,7 @@ Item {
         if (__merginApi.hasAuthData()) {
           authPanel.visible = false
           // filters should be set already
-          __merginApi.listProjects(searchField.text)
+          __merginApi.listProjects("", "created")
```

Comment:
why "created" ? isn't this dependent on the active tab?

Reply:
It should be dependent indeed. However, the same case is `onAuthChanged` when myProjects button is always activated. This was kind of default behaviour to switch to my project first. Have no problem to stay in active tab which makes more sense